src/translate_c.zig

```diff
@@ -1058,6 +1058,13 @@ fn transStmt(
             const compound_literal = @ptrCast(*const clang.CompoundLiteralExpr, stmt);
             return transExpr(c, scope, compound_literal.getInitializer(), result_used);
         },
+        .GenericSelectionExprClass => {
+            const gen_sel = @ptrCast(*const clang.GenericSelectionExpr, stmt);
+            if (gen_sel.isResultDependent()) {
+                return fail(c, error.UnsupportedTranslation, stmt.getBeginLoc(), "TODO implement translation of result-dependent GenericSelectionExpr", .{});
```

Comment:
Based on clang [docs](https://clang.llvm.org/doxygen/classclang_1_1GenericSelectionExpr.html), result-dependent generic selections seem like a C++ feature so we can drop the `TODO` here.

Reply:
Nice, I wasn't sure about that. I removed all the references to `isResultDependent`